src/Indicator.vala

```diff
@@ -86,6 +87,43 @@ public class Network.Indicator : Wingpanel.Indicator {
             popover_widget.closed ();
         }
     }
+
+    private void update_tooltip () {
+        string active_network_name = "some network";
+
+        switch (popover_widget.state) {
+            case Network.State.CONNECTING_WIRED:
+            case Network.State.CONNECTING_WIFI:
+            case Network.State.CONNECTING_MOBILE:
+                display_widget.tooltip_markup = Granite.markup_accel_tooltip ({}, _("Connecting to: " + active_network_name));
```

Comment:
I think let's wrap in quotes instead of using a colon

```suggestion
                display_widget.tooltip_markup = Granite.markup_accel_tooltip ({}, _("Connecting to “%s”".printf (active_network_name)));
```